README.md

```diff
@@ -0,0 +1,51 @@
+# Virtual Pet
+---
+
+Hi! Virtual Pet is a Javascript pet. I'm using this project to practice object oriented programming.
+
+## What will it do?
+
+
+The Virtual Pet will be able to:
+
+* Be given a name
+* Grow older
+* Get hungry
+* Lose fitness
+* Be taken for walks to increase its fitness
+* Be fed to decrease its hunger
+* Be talked to and reply if it needs feeding or walking
+* Die if it becomes too unfit or too hungry
+* Die of old age after a specified number of days
+* Adopt a child
+* Give birth to a child
+
+## Installation
+
+
+* Fork this repository
+* Clone your forked repository
+* Run `npm i` to download relevant dependencies and devDependencies
```

Comment:
why not add the command you would run to clone the repo? You could also add how they run tests as well